core/src/main/java/org/hibernate/ogm/type/impl/BinaryMappedType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.type.impl;
+
+import org.hibernate.MappingException;
+import org.hibernate.engine.spi.Mapping;
+import org.hibernate.ogm.type.descriptor.impl.ByteArrayMappedGridTypeDescriptor;
+import org.hibernate.type.descriptor.java.PrimitiveByteArrayTypeDescriptor;
+
+/**
+ * @author Davide D'Alto
+ */
+public class BinaryMappedType extends AbstractGenericBasicType<byte[]> {
```

Comment:
Is there a reason why it's called Binary? I would have expected PrimaryByteArrayMappedType?